examples/dotnet/SeleniumDocs/BiDi/BrowsingContext/BrowsingContextTest.Create.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.BiDi;
+using OpenQA.Selenium.BiDi.Modules.BrowsingContext;
+using System.Threading.Tasks;
+
+namespace SeleniumDocs.BiDi.BrowsingContext;
+
+partial class BrowsingContextTest
+{
+    [TestMethod]
+    public async Task OpenNewTab()
+    {
+        await using var bidi = await driver.AsBidirectionalAsync();
+
+        var context = await bidi.CreateContextAsync(ContextType.Tab);
+
+        Assert.IsNotNull(context);
+    }
+
+    [TestMethod]
+    public async Task OpenNewWindow()
+    {
+        await using var bidi = await driver.AsBidirectionalAsync();
+
+        var context = await bidi.CreateContextAsync(ContextType.Window);
+
+        Assert.IsNotNull(context);
+    }
+
+    [TestMethod]
+    public async Task OpenTabWithReferenceBrowsingContext()
+    {
+        var context1 = await driver.AsBidirectionalContextAsync();
+
+        var context2 = await context1.BiDi.CreateContextAsync(ContextType.Tab, new() { ReferenceContext = context1 });
```

Comment:
Would be great if `context1` can expose `CreateContextAsync` method, which implicitly sets `ReferenceContext` to the `context1`